lib/device_api/android/adb.rb

```diff
@@ -22,10 +22,10 @@ def self.devices
       end
 
       # Retrieve device state for a single device
-      # @param serial serial number of device
+      # @param qualifier qualifier number of device
```

Comment:
Minor issue - but can you remove 'number' please as it may be an IP (which isn't just numeric!)